### ISSUE TITLE
`require': cannot load such file -- ritm (LoadError)

### ISSUE THREAD

**Issue by Ondrik8** [zt2/sqli-hunter#7]
````root@kali:~/soft/web/sqli-hunter/bin# ruby sqli-hunter.rb 
Traceback (most recent call last):
        7: from sqli-hunter.rb:13:in `<main>'
        6: from sqli-hunter.rb:13:in `require_relative'
        5: from /root/soft/web/sqli-hunter/lib/hunter.rb:6:in `<top (required)>'
        4: from /root/soft/web/sqli-hunter/lib/hunter.rb:15:in `<module:Hunter>'
        3: from /root/soft/web/sqli-hunter/lib/hunter.rb:15:in `require_relative'
        2: from /root/soft/web/sqli-hunter/lib/hunter/proxy.rb:6:in `<top (required)>'
        1: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
/usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require': cannot load such file -- ritm (LoadError)
````
dont start... on kali linux

**Comment by zt2** [comment]
`ritm` is missing, did you Install dependencies?

```
gem install bundler
bundler install
```

**Comment by Ondrik8** [comment]
yes im install  dependencies

сб, 4 апр. 2020 г. в 18:49, ztz <notifications@github.com>:

**Comment by zt2** [comment]
Thanks for your information, I believe this issue has been fixed in version `1.2.1` (https://github.com/zt2/sqli-hunter/releases/tag/1.2.1), please try it again.
